src/plat/win32/detect.c

```diff
@@ -249,15 +261,20 @@ void detect_gpu(void)
 */
 void detect_disk(void)
 {
-	FILE *disk_file;
-	char drive[MAX_STRLEN];
 	char *disk_unit;
 	long long totalBytes, freeBytes, usedBytes, disk_used, disk_total,
 		disk_percentage;
 
+#ifdef __MINGW32__
+	LPCTSTR drive = "C:\\";
```

Comment:
Yeah, assuming `C:\` is perfectly fine. My understanding is that Windows assumes this as well, so we're no more or less correct than the rest of the system.